src/js/WebParts/BenefitsOverview/index.tsx

```diff
@@ -205,7 +205,7 @@ export default class BenefitsOverview extends BaseWebPart<IBenefitsOverviewProps
                 startIndex: groupNames.indexOf(name, 0),
                 count: [].concat(groupNames).filter(n => n === name).length,
                 isCollapsed: false,
-                isShowingAll: true,
+                isShowingAll: false,
```

Comment:
@Remi749 We could set isShowingAll to true if `count` of the group is equal to the total size.